Library/Homebrew/utils/analytics.sh

```diff
@@ -1,30 +1,42 @@
 setup-analytics() {
-  # User UUID file. Used for Homebrew user counting. Can be deleted and
-  # recreated with no adverse effect (beyond our user counts being inflated).
+  HOMEBREW_GIT_CONFIG_FILE="$HOMEBREW_REPOSITORY/.git/config"
+
+  # User UUID file. This file has been deprecated in favor of
+  # homebrew.analyticsuuid in $HOMEBREW_REPOSITORY/.git/config. If detected,
+  # the UUID will be written to the aforementioned git config file, and the
+  # original UUID file will be removed.
   HOMEBREW_ANALYTICS_USER_UUID_FILE="$HOME/.homebrew_analytics_user_uuid"
```

Comment:
This variable is only used in `setup-analytics` if I'm not mistaken. IMHO, this is good reason to shorten it (drop at least the `HOMEBREW_` prefix) and to make it a local variable (this would also mean make it lower-case) to reduce pollution of the global scope.